src/style.css

```diff
@@ -1 +1,75 @@
 /* Styles go here */
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  color: var(--primary-color);
```

Comment:
Declare custom properties for colors at the beginning of the file, preferably in a :root selector instead of within the body selector.